defaults/main.yml

```diff
@@ -143,7 +137,7 @@ traefik_docker_exposed_by_default: False
 traefik_docker_bind_port_ip: False
 
 # @var traefik_docker_network_name:description: Docker network used by docker provider
-traefik_docker_network_name: traefik
+# @var traefik_docker_network_name:example: $ "traefik"
```

Comment:
I would prefer the general default of `traefik` as everything is built around that.